module-ballerina-openapi/CompilerPlugin.toml

```diff
@@ -3,7 +3,7 @@ id = "openapi-tools"
 class = "io.ballerina.openapi.validator.OpenAPIValidatorPlugin"
 
 [[dependency]]
-path = "../openapi-validator/build/libs/openapi-validator-@project.version@.jar"
+path = "../openapi-validator/build/libs/openapi-validator-2.1.1-SNAPSHOT.jar"
 groupId = "ballerina"
 artifactId = "openapi"
-version = "@project.version@."
+version = "2.1.1-SNAPSHOT"
```

Comment:
we need to keep @project.version. Need to fix in other places too

Reply:
I think this is from the auto-commit. I will replace in all the relevant places.

Reply:
Shall I send the changes directly to `master` instead of the `java21` branch? @warunalakshitha

Reply:
+1